File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        l, r = 0, n

        res = []

        while r < len(nums):
            res.append(nums[l])
            res.append(nums[r])
            l += 1
            r += 1

        return res

File: 1470-shuffle-the-array/1470-shuffle-the-array.go
func shuffle(nums []int, n int) []int {
    var a, b = 0, n
    res := []int{}

    for {
        res = append(res, nums[a], nums[b])
        a++
        b++
        if b == len(nums) {
            break
        }
    }
    
    return res
}